Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -6941,7 +6941,6 @@ class X86Assembler {
                 writer.threeBytesVexNds(simdPrefix, VexImpliedBytes::TwoBytesOp, vexW, (RegisterID)0, (RegisterID)0, (RegisterID)0);
             else
                 writer.twoBytesVex(simdPrefix, (RegisterID)0, (RegisterID)0);
-            writer.twoBytesVex(simdPrefix, (RegisterID)0, (RegisterID)0);
```

Comment:
Fix vzeroupper generation. I found a bug once I started using vzeroupper (but not using it right now).